resources/aws/securitygroup.go

```diff
@@ -0,0 +1,77 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+var (
+	portsToOpen []int = []int{
+		// SSH
+		22,
+		// Insecure apiserver
+		80,
+		// apiserver
+		443,
+		// etcd
+		2379,
+		// Insecure apiserver
+		8080,
+	}
+)
+
+type SecurityGroup struct {
+	Description string
+	GroupName   string
+	VpcID       string
+	name        string
+	AWSEntity
+}
+
+func (s *SecurityGroup) CreateIfNotExists() (bool, error) {
+	return false, microerror.MaskAny(notImplementedMethodError)
+}
+
+func (s *SecurityGroup) openPort(port int) error {
+	if _, err := s.Clients.EC2.AuthorizeSecurityGroupIngress(&ec2.AuthorizeSecurityGroupIngressInput{
+		CidrIp:     aws.String("0.0.0.0/0"),
```

Comment:
I think if we leave it like this then we should definitely add an issue to remember to change it in the future.